app/views/about/technicaldetails.html.erb

```diff
@@ -16,6 +16,8 @@
 
 </ul>
 
+<p>Additional content in the <%= link_to "Collections Hub prototype", "https://cdrh.reclaim.hosting/omekas/s/apdp/" %>
```

Comment:
Says this page is still under maintenance - going to check with Ella and Karin to make sure it's the right URL

Reply:
They'd like us to skip this for now - there was an error updating the site and it's stuck in maintenance mode. If you want to go ahead and delete the whole line, that would be great; I think they've changed their minds about how they'll eventually link to the prototype, when they eventually do